binaries/record_function_benchmark.cc

```diff
@@ -19,10 +19,10 @@ const float kLowSamplingProb = 0.0001;
 
 void addTestCallback(
     double sampling_prob = 1.0,
-    std::function<std::unique_ptr<at::ObserverContext>(const at::RecordFunction&)> fn =
-        [](const at::RecordFunction&) { return nullptr; }) {
+    at::RecordFunctionCallback::StartCallback fn =
+        [](const at::RecordFunction&) -> std::unique_ptr<at::ObserverContext> { return nullptr; }) {
```

Comment:
Why do we need a non-trivial initializer here?
```suggestion
    at::RecordFunctionCallback::StartCallback fn = at::RecordFunctionCallback::StartCallback()
```

Reply:
I just left it the way the existing code had it, don't think it's expensive either way because it's a constant expression (address of a function)